src/server/pretalx/PretalxAPIPosters.ts

```diff
@@ -0,0 +1,12 @@
+import postersData from "../../../data/posters.json";
+import { pick } from "../../utils/pick";
+
+/**
+ * A Poster that is associated with a Poster session.
+ */
+export type PretalxAPIPoster = typeof ALL_POSTERS[number];
+
+export const ALL_POSTERS = postersData.map((poster) => ({
+  ...pick(poster, ["title", "abstract", "description", "day", "pdflink", "speakers"]),
```

Comment:
Does this not cause a type error? We don't actually have the pdflink field defined in the `posters.json` file.